libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/DialogManager.cs

```diff
@@ -243,5 +246,16 @@ public async Task<DialogManagerResult> OnTurnAsync(ITurnContext context, Cancell
 
             return new DialogManagerResult() { TurnResult = turnResult };
         }
+
+        public void RegisterDefaultLanguagePolicy(Dialog dialog)
+        {
+            if (dialog is AdaptiveDialog adialog && adialog.Generator != null)
+            {
+                if (adialog.Generator is ResourceMultiLanguageGenerator rGenerator)
+                {
+                    TurnState.Set<LanguagePolicy>(rGenerator.LanguagePolicy);
+                }
+            }
+        }
```

Comment:
do this as an extension method on DialogManager.cs
dm.TurnState.Set<LanguagePolicy>(new LanguagePolicy(defaultLanguage));

Reply:
an extension looks clean, but the question like above is how i get the timing to call this extension, i want share the policy that configured in root dialog.